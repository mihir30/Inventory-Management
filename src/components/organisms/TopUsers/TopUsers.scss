.reports-table-tbody {
    height: 545px;
    overflow-y: auto;
    width: 100%;
}

.reports-table-thead,
.reports-table-tbody,
.reports-table-tr,
.reports-table-td,
.reports-table-th {
    display: block;font-size:14px;
}
.wrench-icon {
    margin-right: 1rem;
    color: grey;
}
.reports-table-tbody,
.reports-table-td {
    display: inline-block;
}

.reports-table-tbody,
.reports-table-th,
.reports-table-thead > tr > th {
    float: left;
    position: relative;
    background-color: #f4f5f9;
}

.table-header-topusers {
    padding-left: 0px;
    height: 43px;
    background: #f4f5f9;
    color: #313131;
    font-weight: 500;
    letter-spacing: .2px;
    clear: both;
    overflow: hidden;
    border-bottom: 1px solid #5ccce7;
    width:93vw;
}

.table-container-reports{
    margin-left:4px;
    font-size:14px;
}
