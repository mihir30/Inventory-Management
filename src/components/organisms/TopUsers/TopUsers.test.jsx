import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDom from "react-dom";
import TopUsers from './TopUsers';

describe("Top users Component", () => {
    

    it('renders the component properly', () => {
        const divEl = document.createElement("div");
        ReactDom.render(<BrowserRouter><TopUsers></TopUsers></BrowserRouter>, divEl); 
    });


    it('should have the right top users component snapshot', () => {
        const tree = renderer.create().toJSON();
        expect(tree).toMatchSnapshot();
    })

});