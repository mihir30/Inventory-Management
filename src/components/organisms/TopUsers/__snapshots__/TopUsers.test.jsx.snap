// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top users Component should have the right top users component snapshot 1`] = `null`;
